src/runtime-rs/Makefile

```diff
@@ -17,6 +17,9 @@ CONTAINERD_RUNTIME_NAME = io.containerd.kata.v2
 
 include ../../utils.mk
 
+# image service crates need to compile target to x86_64-unknown-linux-gnu
+override LIBC = gnu
```

Comment:
Doesn't nydus support musl?

https://github.com/dragonflyoss/image-service/blob/5c402656af2947ffd234980545c64dad7a1fc33a/Makefile#L6

Reply:
Thanks ! it's been fixed now. At first, I thought the users will be struggling with installing openssl-devel to compile openssl so I choose `gnu` as the target.  However, I find `openssl = { version = "0.10.35", features = ["vendored"] }` for statically compile.